core/src/com/unciv/logic/map/TileMap.kt

```diff
@@ -129,6 +127,31 @@ class TileMap {
                     }
                 }.filterNotNull()
 
+    private fun getIfTileExistsOrNull(x: Int, y: Int) : TileInfo? {
+        if (contains(x, y))
+            return get(x, y)
+
+        if (!mapParameters.worldWrap)
+            return null
+
+        var radius = mapParameters.size.radius
+        if (mapParameters.shape == MapShape.rectangular)
+            radius = HexMath.getEquivalentRectangularSize(radius).x.toInt() / 2
+
+        if (!contains(x, y)) { //tile is outside of the map
```

Comment:
Surely this would always be 'true' since we check contains(x,y) at the first line in the function?